app/services/promo/peer_to_peer_registration.rb

```diff
@@ -0,0 +1,49 @@
+class Promo::PeerToPeerRegistration < BaseApiClient
```

Comment:
I think this might look better in something that's similar to the new API client structure we have

Like this
https://github.com/brave-intl/publishers/blob/07702bfbe7df53448496b1f0f5bd0851ceac75da/app/services/promo/models/owner_state.rb#L37-L43

Or 

https://github.com/brave-intl/publishers/blob/staging/app/services/promo/models/reporting.rb#L9-L28